test/PublishCommand.js

```diff
@@ -907,6 +944,84 @@ describe("PublishCommand", () => {
         }
       }));
     });
+
+    /** =========================================================================
+    * INDEPENDENT - CD VERSION - PRERELEASE
+    * ======================================================================= */
+
+    it("should bump to prerelease versions with --cd-version=prerelease --prerelease-id=foo", (done) => {
```

Comment:
These are great tests, what happens when you bump a non-canary `prepatch` with another `prepatch`?

I would expect `1.0.0-beta.0` bumps to `1.0.0-beta.1`, but I don't see any tests to that effect.